package.json

```
@@ -39,7 +39,7 @@
     "test": "node ./out/test/runTest.js",
     "format": "prettier 'src/**/*.ts' --write",
     "lint": "eslint src --ext ts --fix",
-    "pretest": "npm run test-compile && npm run lint"
+    "pretest": "npm run compile:extension"
```

```suggestion
    "pretest": "npm run compile"
```
You can change it to this so that it compiles react too

---

This doesn't work for me because both `build:v3` and `postbuild:v3` reference `couchbase-cloud` as an external project, which I don't have.